lib/service/config.go

```diff
@@ -13,7 +13,9 @@ type Config struct {
 	JWTRefreshTokenExpiry int    `envconfig:"JWT_REFRESH_EXPIRY" default:"604800"` // in seconds, default 7 days
 	JWTAccessTokenExpiry  int    `envconfig:"JWT_ACCESS_EXPIRY" default:"172800"`  // in seconds, default 2 days
 	LNDAddress            string `envconfig:"LND_ADDRESS" required:"true"`
-	LNDMacaroonHex        string `envconfig:"LND_MACAROON_HEX" required:"true"`
+	LNDMacaroonFile       string `envconfig:"LND_MACAROON_FILE"`
+	LNDCertFile           string `envconfig:"LND_CERT_FILE"`
```

Comment:
I was planning to implement these two as well. It is a perfect candidate to be extracted into a separate PR.

Reply:
Extracted into https://github.com/getAlby/lndhub.go/pull/218